pmd-java/src/main/java/net/sourceforge/pmd/lang/java/symboltable/ClassScope.java

```diff
@@ -295,15 +294,15 @@ private void matchMethodDeclaration(JavaNameOccurrence occurrence,
      */
     private MethodNameDeclaration createBuiltInMethodDeclaration(final String methodName,
             final String... parameterTypes) {
-        ASTMethodDeclaration methodDeclaration = new ASTMethodDeclaration(JavaParserTreeConstants.JJTMETHODDECLARATION);
+        ASTMethodDeclaration methodDeclaration = new ASTMethodDeclaration(0);
```

Comment:
Yes. But I didn't replace the ID with a valid one because anyway this piece of code is going away. Nodes should not be created outside of the ast package